plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueInputConverter.java

```diff
@@ -64,12 +66,16 @@ public static DatabaseInput convertDatabase(Database database)
 
     public static TableInput convertTable(Table table)
     {
+        TableInput input = new TableInput();
+        Map<String, String> tableParameters = table.getParameters();
         Optional<String> comment = Optional.ofNullable(table.getParameters().get(TABLE_COMMENT));
-        Map<String, String> tableParameters = table.getParameters().entrySet().stream()
-                .filter(entry -> !entry.getKey().equals(TABLE_COMMENT))
-                .collect(toImmutableMap(Map.Entry::getKey, Map.Entry::getValue));
+        if (!(isTrinoView(table) || isTrinoMaterializedView(table))) {
```

Comment:
nit: reduce number of braces


```suggestion
        if (!isTrinoView(table) && !isTrinoMaterializedView(table)) {
```